bindings/rust/standard/bench/src/harness/mod.rs

```diff
@@ -381,6 +425,13 @@ mod tests {
 
                         assert!(conn_pair.negotiated_tls13());
                         assert_eq!(cipher_suite, conn_pair.get_negotiated_cipher_suite());
+
+                        // read in "application data" handshake messages.
+                        // "NewSessionTicket" in the case of resumption
+                        let err = conn_pair.client_mut().recv(&mut [0]).unwrap_err();
+                        assert_eq!(&err.to_string(), "blocking");
```

Comment:
I see you doing this twice, and resumption tests will likely need it more-- worth a named method?